app/models/subject.rb

```diff
@@ -11,8 +11,4 @@ def self.sorted_by_number
   def self.with_assessments
     joins(:assessments).distinct
   end
-
-  def to_s
```

Comment:
We're still relying on this elsewhere, so I think we still need this method. This is causing the test failures.